src/test/packages/gitrepo/zarf.yaml

```diff
@@ -7,10 +7,10 @@ metadata:
   version: 0.0.1
 
 components:
-  - name: dubbd-repo
+  - name: mattermost
     required: true
     repos:
-      - https://github.com/defenseunicorns/uds-package-dubbd
+      - https://github.com/defenseunicorns/uds-package-mattermost@v10.2.0-uds.1
```

Comment:
I might just be missing something but I'm a bit suspicious of this change given the test file name. Zarf has a feature to push a gitrepo to a git server "in the airgap" (similar to how it pushes images). I think this test was meant to validate that functionality, and I'm not sure `uds-package-mattermost` uses that?

At the same time, UDS is pretty opinionated to not ever use the gitrepo feature so there might be a route for dropping this test (although I'd be a bit wary of doing that).

Reply:
I believe the intent of this test was/is "can we create a bundle that uses a `zarf.yaml` which contains a reference to another package via `https://`. I don't think this test is meant to verify the behavior you describe (there are other tests around pushing).

The reason for this change was Zarf was bailing trying to find the `dubbd` package. I believe it's archived and maybe that's a problem?

Reply:
Yep - totally misread this the first time around, this pulls the repo NOT the zarf package.